django/db/models/fields/reverse_related.py

```diff
@@ -176,9 +177,6 @@ def get_accessor_name(self, model=None):
             return self.related_name
         return opts.model_name + ('_set' if self.multiple else '')
 
-    def get_cache_name(self):
```

Comment:
`ForeignObjectRel` has a name property but I've noticed it does not do the same as `get_accessor_name()`, will investigate. Good catch!.